test/contracts/promises/three-contracts/third-contract/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+#![no_std]
+#![no_main]
+#![allow(unused_attributes)]
+#![feature(lang_items)]
```

Comment:
I think `#[no_std]` is the only one needed of all.

Reply:
Removed the rest.